java/app-encryption/src/it/java/com/godaddy/asherah/ConfigReader.java

```diff
@@ -0,0 +1,30 @@
+package com.godaddy.asherah;
+
+public class ConfigReader {
+
+  private String metastoreType;
+  private String metastoreJdbcUrl;
```

Comment:
Add params for username, password, connectionpool size (like the existing test-app) for more flexibility.